src/hclib.c

```diff
@@ -68,9 +79,10 @@ hclib_ddf_t *hclib_async_future(futureFct_t fp, void *arg,
 #define DEBUG_FORASYNC 0
 
 forasync1D_task_t * allocate_forasync1D_task() {
-    forasync1D_task_t * forasync_task = (forasync1D_task_t *) malloc(
+    forasync1D_task_t *forasync_task = (forasync1D_task_t *) malloc(
             sizeof(forasync1D_task_t));
     assert(forasync_task && "malloc failed");
```

Comment:
Sorry if I left some places still using assert rather than HC_ASSERT. This should be changed at all places.